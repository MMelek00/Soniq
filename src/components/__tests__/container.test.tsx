import 'react-native';
import React from 'react';

import renderer from 'react-test-renderer';
import Container from '../container';
import GenericText from '../genericText';
describe('Container', () => {
  it('renders correctly with Error', () => {
    const tree = renderer.create(
      <Container>
        <GenericText size={32} text="Normal Text" />,
      </Container>,
    );
    expect(tree).toMatchSnapshot();
  });
});
