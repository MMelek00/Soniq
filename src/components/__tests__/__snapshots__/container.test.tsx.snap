// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders correctly with Error 1`] = `
<RCTSafeAreaView
  style={
    [
      {
        "backgroundColor": "#e5e5e5",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
      },
    ]
  }
>
  <Text
    size={32}
    style={
      [
        {
          "color": "black",
          "fontSize": 32,
          "fontWeight": "normal",
          "marginBottom": 10,
          "marginLeft": 0,
          "marginRight": 0,
          "marginTop": 10,
          "textalign": "'center'",
        },
      ]
    }
  >
    Normal Text
  </Text>
  ,
</RCTSafeAreaView>
`;
